.github/workflows/main.yml

```diff
@@ -0,0 +1,58 @@
+name: Main
+
+on:
+  push:
+    branches:
+      - master
+      - spec_live
+  pull_request:
+    branches:
+      - master
+      - spec_live
```

Comment:
I don't think you need this. We won't be creating pull requests against the `spec_live` branch